src/pages/AddServers/SelectInstallOS.js

```diff
@@ -0,0 +1,125 @@
+// (c) Copyright 2018 SUSE LLC
+/**
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+import React from 'react';
+import BaseUpdateWizardPage from '../BaseUpdateWizardPage.js';
+import { LoadingMask } from '../../components/LoadingMask.js';
+import { translate } from '../../localization/localize.js';
+import { ErrorMessage } from '../../components/Messages.js';
+import { YesNoModal } from '../../components/Modals.js';
+import SelectServersToProvision from '../SelectServersToProvision.js';
+import { postJson } from '../../utils/RestUtils.js';
+
+// This page allow the user to select newly added compute
+// servers to install SLES on them
+class SelectInstallOS extends BaseUpdateWizardPage {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      // show confirm
+      showInstallConfirmModal: false,
+      // error msg
+      passphraseError: undefined,
+      // disable the next button when it is not valid
+      isInvalid: true
+    };
+  }
+
+  componentWillReceiveProps(newProps) {
+    let isInvalid =
+      !newProps.operationProps.selectedToInstallOS ||
```

Comment:
Instead of using this same pattern everywhere of checking that the property exists and that its length is 0 (or not 0), please consider using `isEmpty` from lodash. It would make it much easier to understand, e.g.: 
```
  const isInvalid = 
    isEmpty(newProps.operationProps.selectedToInstallOS) ||
    isEmpty(newProps.operationProps.osInstallPassword) ||
    (isEmpty(newProps.operationProps.sshPassphrase) && newProps.operationProps.sshPassphraseRequired);
```
I also see this in other places in this PR

Reply:
k will update